### DISCUSSION TITLE
[React] workflows가 생성될 수 있는 브랜치에 관한 질문

### DISCUSSION THREAD

**Discussion by Yumin22224** [wafflestudio/seminar-2023#176] [Q&A] [answered]
HW1과제 맨 마지막 부분에서, .github/workflows/ 에 테스트 CI 파일을 올리기 위해 Actions 탭에서 New workflow 버튼을 누른 뒤 playwright.yml 파일을 복사 및 붙여넣기 해서 생성하려고 했습니다. 이때 브랜치 선택이 불가하고 main에서밖에 생성이 되지 않아 일단 main에 해당 파일을 올려두었습니다.
과제를 하면서 main 브랜치는 건드리지 않으려고 했는데, hw1에서 똑같이 New workflow 버튼을 눌러봐도 브랜치가 main으로밖에 되어있지 않아서, 다른 브랜치에서도 workflow를 만들 수는 없는 건지 질문 드립니다!

**Comment by woohm402** [comment, marked as answer]
github web에서 workflow를 생성하는 방식은 제가 몇 번 사용해보진 않았는데, workflow 파일을 생성해서 main에 바로 푸시하거나 / main으로 PR을 자동으로 생성하거나 이 정도 옵션이 있었던 것 같습니다.

원래 저희가 의도한 건 로컬에서 `.github/workflows` 폴더 만들고 거기에 파일 만든 다음 푸시하는 거였습니다!

https://github.com/wafflestudio/seminar-2023/assets/39977696/643507f9-8d7f-4a8e-99a7-4ab4ba527ded

영상에서는 github 에 연결하지 않아서 git push 가 실패했는데, 원래대로면 됐을거예요

굳이 깃헙 웹 이용할 거 없이 그냥 이렇게 하면 편하고 깔끔합니다 :)
